### PR TITLE
Dockerized Build

### PR DESCRIPTION
**Pull request by thefringeninja**
The goal of this PR is to deprecate `eventstore-docker` and instead produce the container from the main repository.

### CODE REVIEW

File: src/EventStore.ClusterNode/Program.cs [hunk 1]
```diff
@@ -124,15 +124,15 @@ public class Program : ProgramBase<ClusterNodeOptions> {
 			_host = new WebHostBuilder()
 				.UseKestrel(o => {
 					o.Listen(opts.IntIp, opts.IntHttpPort);
-					o.Listen(opts.ExtIp, opts.ExtHttpPort,
-						listenOptions => listenOptions.UseHttps());
+					o.Listen(opts.ExtIp, opts.ExtHttpPort, listenOptions => listenOptions.UseHttps(_node.Certificate));
```
[the remark this answers is not in the document]
Comment (thefringeninja) [reply src/EventStore.ClusterNode/Program.cs]: See #2083

File: src/EventStore.ClusterNode/Program.cs [hunk 2]
```diff
@@ -320,21 +320,19 @@ public class Program : ProgramBase<ClusterNodeOptions> {
...
+				builder.WithServerCertificateFromStore(location, name, options.CertificateSubjectName,
+					options.CertificateThumbprint);
+			} else if (!string.IsNullOrWhiteSpace(options.CertificateStoreName)) {
+				var name = GetCertificateStoreName(options.CertificateStoreName);
+				builder.WithServerCertificateFromStore(name, options.CertificateSubjectName,
+					options.CertificateThumbprint);
+			} else if (options.CertificateFile.IsNotEmptyString()) {
+				builder.WithServerCertificateFromFile(options.CertificateFile, options.CertificatePassword);
+			} else
+				Log.Warn("No server certificate specified.");
```
[the remark this answers is not in the document]
Comment (thefringeninja) [reply src/EventStore.ClusterNode/Program.cs]: It's not precisely required, if you just do `UseHttps()` then the default kestrel dev cert is used. That's fine in dev but production maybe not. How to detect that tho...

**Comment by thefringeninja** [comment]
@pgermishuys  split the PR like you asked. however now there is a new problem. On my machine at least when I try to run I get 
```
A fatal error occurred. The required library libhostfxr.so could not be found.
If this is a self-contained application, that library should exist in [/opt/eventstore/].
If this is a framework-dependent application, install the runtime in the global location [/usr/share/dotnet] or use the DOTNET_ROOT environment variable to specify the runtime location or register the runtime location in [/etc/dotnet/install_location].
```
This happened even on the outdated branch...

**Comment by pgermishuys** [comment]
Closing in favor of https://github.com/EventStore/EventStore/pull/2087

**Pull request closed without merging**